tests/functional/test_sentry.py

```diff
@@ -71,25 +74,25 @@ def test_directory_stat(self):
     def test_directory_listing(self):
         path = '/tmp/amulet-test'
         self.assertEqual(
-            self.pictor.directory_listing(path), {
+            self.nagios.directory_listing(path), {
                 'files': ['test-file'],
                 'directories': ['test-dir'],
             },
         )
 
     def test_relation(self):
-        pictor_info = self.pictor.relation(
+        nagios_info = self.nagios.relation(
             'website', 'haproxy:reverseproxy')
         for key in ['hostname', 'port', 'private-address']:
-            self.assertTrue(key in pictor_info)
+            self.assertTrue(key in nagios_info)
 
         haproxy_info = self.haproxy.relation(
-            'reverseproxy', 'pictor:website')
+            'reverseproxy', 'nagios:website')
         self.assertEqual(list(haproxy_info.keys()), ['private-address'])
 
     def test_run(self):
         self.assertEqual(
-            self.pictor.run('echo hello'),
+            self.nagios.run('echo hello'),
```

Comment:
Would be nice to run some sentry commands against rsyslog-forwarder to confirm they work properly for subs.